table_schema_cache.go

```diff
@@ -268,23 +268,8 @@ func (t *TableSchema) paginationKeyColumn(cascadingPaginationColumnConfig *Casca
 	return paginationKeyColumn, err
 }
 
-/*
-GetPKColumn is a method originally belonging to go-mysql/schema.Table
-and stands for: `Get Primary Key Column`
-
-As `TableSchema` embeds `schema.Table`, previously the method call to
-`TableSchema` would delegate automatically to the method for
-`schema.Table`. This was in spirit with the initial assumption that the
-column to iterate over in any given table is the Primary Key Column.
-
-In order to accommodate user provided pagination columns through
-configuration in the least painful way, this method has been
-reimplemented. Now this method stands for: `Get Pagination Key Column`.
-
-The param `index` currently serves no purpose but to maintain the same
-method signature as the original method.
-*/
-func (t *TableSchema) GetPKColumn(index int) *schema.TableColumn {
+// GetPaginationColumn retrieves PaginationKeyColumn
+func (t *TableSchema) GetPaginationColumn(index int) *schema.TableColumn {
```

Comment:
The `index` argument is not necessary afaict.

Reply:
Correct. I am tackling changes one commit at a time.